packages/ffe-form-react/src/PhoneNumberField.js

```diff
@@ -0,0 +1,82 @@
+
+export default function PhoneNumberField({
+                               phone,
+                               countryCode,
+                               onChange,
+                               onBlur,
+                               locale,
+                               disabled,
+                               showValidationError,
+                               className
+                           }) {
+
+    const supportedLocales = ['nb', 'nn', 'en'];
+    const text = i18n[supportedLocales.indexOf(locale) !== -1 ? locale : 'nb'];
+
+    function onPhoneChange(e) {
+        onChange(countryCode, e.target.value);
+    }
+
+    function onCountryCodeChange(e) {
+        onChange(e.target.value, phone);
+    }
+
+    function handleBlur(e) {
+        if (e.target.id !== 'country-code' || e.target.value.length === 0) {
+            onBlur(e);
+        }
+    }
+
+    return (
+        <div className={`ffe-input-group ${className ? className : 'phone-number-field'}`}>
+            <div className="phone-number-field">
+                <div className="phone-number-field__country-code">
+                    <label className="ffe-form-label" htmlFor="country-code">{text.COUNTRY_CODE}</label>
+                    <div className="phone-number-field__input-group">
+                        <span className="phone-number-field__plus">+</span>
+                        <input
+                            id="country-code"
+                            maxLength="3"
+                            className="ffe-input-field phone-number-field__country-code-input"
+                            type="tel"
+                            disabled={disabled}
+                            value={countryCode}
+                            aria-invalid={showValidationError}
+                            onChange={onCountryCodeChange}
+                            onBlur={handleBlur}
+                        />
+                    </div>
+                </div>
+                <div className="phone-number-field__number">
+                    <label className="ffe-form-label" htmlFor="phone-input">{text.PHONE_NUMBER}</label>
+                    <input
+                        id="phone-input"
+                        maxLength="14"
+                        type="tel"
+                        className="ffe-input-field phone-number-field__phone-input"
+                        onChange={onPhoneChange}
+                        onBlur={onBlur}
+                        value={phone}
+                        aria-invalid={showValidationError}
+                        disabled={disabled}
+                    />
+                </div>
+            </div>
+        </div>
+    );
+}
+
+PhoneNumberField.propTypes = {
+    phone: string.isRequired,
+    countryCode: string.isRequired,
+    onChange: func.isRequired,
+    onBlur: func.isRequired,
+    locale: string,
+    disabled: bool,
+    showValidationError: bool,
+    className: string
+};
```

Comment:
I would prefer it if neither of these fields were required. 

- The `phone` and `countryCode` props can be skipped, so you don't require this to be a [controlled component](https://reactjs.org/docs/forms.html#controlled-components).
- The `onChange` prop could be added further up the component tree. On our team, we use a validation technique that does just this.
- The `onBlur` prop is probably not required for all usecases.

Instead of requiring `onChange` and `onBlur` (or any function that is not required to make stuff actually work), you can pass in no-op functions in the static `defaultProps` field, like this:

```js
PhoneNumber.defaultProps = {
  onChange: f => f,
  onBlur: f => f,
};
```

This way, you can call them in event handlers, and assume that they're always callable.